webapp/maps_heatmaps/src/index.tsx

```diff
@@ -92,8 +93,9 @@ function initMap() {
     data: [],
     map: map,
   });
-  map.addListener("center_changed", () => mapChanged());
+  map.addListener("drag", () => mapChanged());
   map.addListener("zoom_changed", () => mapChanged());
+  google.maps.event.addListenerOnce(map, "center_changed", () => mapChanged());
```

Comment:
Is this the fix regarding the InfoPanel? Nice

Reply:
Yes this is the change to fix the infoWindow.